gcsio/src/test/java/com/google/cloud/hadoop/gcsio/integration/GoogleCloudStorageTestHelper.java

```diff
@@ -256,12 +264,23 @@ public TestBucketHelper(String bucketPrefix) {
 
     private static String makeBucketName(String prefix) {
       String username = System.getProperty("user.name", "unknown").replaceAll("[-.]", "");
-      username = username.substring(0, min(username.length(), 9));
+      // Only first 6 characters of username is considered.
+      username = username.substring(0, min(username.length(), 6));
```

Comment:
Can we keep here 9 (becuase if we take only 6 then there could be confusion like deepan(for deepanshu) and deepen(for deependra)) as before and change the 48 check below to lower number?

Reply:
Considering upto 8 chars of username and remaining are being filled with unique UUID.
There may be cases all 8 characters are same for 2 different users, But UUID ensures prefix is unique.